README.md

```diff
@@ -211,6 +211,10 @@ passed to Lombok's `@Builder` annotation), then the return type of that method i
 the same `@CalledMethods` annotation as the receiver of `build()`, using the same rules as above.
 * A `@This` annotation is placed on the return type of each setter in the builder's implementation.
 
+You can disable the framework supports by specifying them in a comma-separated list to the 
+command-line flag `disabledFrameworkSupports`.  For example, to disable both Lombok and AutoValue supports,
+use `diabledFrameworkSupports=AutoValue,Lombok` . 
```

Comment:
There is a typo in the name of the command line option.

Also, this is an option to an annotation processor, so just supplying `-disabledFrameworkSupports=AutoValue`, for example, will not work; it has to be `-AdisabledFrameworkSupports=AutoValue`.